web/src/components/CategoryListItem.tsx

```diff
@@ -23,6 +23,7 @@ const CategoryThumbnail = styled.img`
   padding: 0 5px;
   flex-shrink: 0;
   object-fit: contain;
+  filter: ${props => (props.theme.isContrastTheme ? 'brightness(0) invert(1)' : 'none')};
```

Comment:
❓ Do we need the `brightness(0)` as well? Looks to me like `invert(1)` should already be enough?

Reply:
good catch! :)